leaf-ui/rappid-extensions/ActorInspector.js

```diff
@@ -50,12 +52,46 @@ var ActorInspector = Backbone.View.extend({
             // Do not allow special characters in names, replace them with spaces.
             text = text.replace(/[^\w\n-]/g, ' ');
 
-            this.cell.attr({ '.name': { text: text } });
-            this.actor.nodeName = text;
+            this.model.attr({ '.name': {text: text }});
+            this.model.set('actorName', text);
 
         },
-      clear: function(){
-            this.$el.html('');
+        /**
+         * Removes the view so we don't have multiple ones in the sidebar
+         */
+        removeView: function(){
+            this.remove();
+        },
+        /**
+         * Changes the line that distinguishes the type of actor 
+         */
+        updateType: function(){
+            this.model.set('actorType', this.$('#actor-type-ID').val());
+        },
+        /**
+         * Changes the line that distinguishes the type of actor 
+         */
+        changeLine: function() {
+            if (this.model.get('actorType') == 'G') {
```

Comment:
Optional: Maybe a variable for this.model.get('actorType') instead of calling it 3 times?